tasks/apt.yml

```diff
@@ -1,6 +1,10 @@
 - name: Update APT package cache
   apt: update_cache=yes
   changed_when: false
+  retries: 5
```

Comment:
@dwang sorry, i am not really aware of what was causing the failure and how does this help, can you please share your research? 

Thanks.

Reply:
@atb00ker There was not a failure directly related to the playbook, but with `ansible-lint`. It was complaining that there was not a retry in the case that remote tasks failed.
For example:
```
[405] Remote package tasks should have a retry
/home/travis/.ansible/roles/role_under_test/tasks/apt.yml:1
Task/Handler: Update APT package cache
```

With the PR, I fixed some more `ansible-lint` errors. I'm not sure why the lint errors weren't present before, but it was causing builds for master and other PRs to fail.

Reply:
Alright thanks. :+1:

Reply:
good call :+1: